apps/ui/lib/lens/list/Interleaved/component.tsx

```diff
@@ -139,6 +139,20 @@ export const InterleavedList = (props: Props) => {
 			startReached={loadMoreContracts}
 			itemContent={itemContent}
 			overscan={10}
+			components={{
```

Comment:
What is this Footer component for?

Reply:
Adds a last "empty" element on the virtuoso list, which have the same purpose as the margin-bottom.

We cannot easily attach the margin-bottom to any of the internal virtuoso divs and doing on the wrapper reduces the scrollview.

Next step for this footer is to add a spinner while loading and a "end of scroll" message (or image)